io.github.multiscrobbler.yml

```diff
@@ -0,0 +1,86 @@
+app-id: io.github.multiscrobbler
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: multiscrobbler
+finish-args:
+  - --share=network
```

Comment:
so this display nothing?

Reply:
Correct. There is output to the command line when run in the foreground but the main ui is browsed-based from a web server. I couldn't find any examples of other flatpak apps that don't have a ui...The main purpose for the flatpak app is to allow [MPRIS scrobbling](https://github.com/FoxxMD/multi-scrobbler/blob/0.4.4/docs/configuration.md#mpris) by using the flatpak app as a [background service](https://github.com/FoxxMD/multi-scrobbler/blob/develop/docs/service.md)

Reply:
this is not the ideal use case for flatpak.